src/client/components/DecksPickBreakdown.tsx

```diff
@@ -167,8 +171,8 @@ const CubeBreakdown: React.FC<BreakdownProps> = ({ draft, seatNumber, pickNumber
   const onPickClick = (packIndex: number, pickIndex: number) => {
     let picks = 0;
     for (let i = 0; i < packIndex; i++) {
-      if (draft.InitialState?.[0]?.[i]?.cards?.length) {
-        picks += draft.InitialState[0][i].cards.length;
+      if (picksList[i]?.length) {
+        picks += picksList[i].length;
```

Comment:
Only functional change (the rest is prettier). Since draft.InitialState only exists for Cube Cobra drafts it wasn't adding to the index for Draftmancer packs. In the refactored code both types of drafts have picksList as the array of packs the current drafter did, so equivalent to `draft.InitialState?.[0]?.[i]`